src/index.html

```diff
@@ -5,9 +5,49 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin = "anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page">
+    <div class="card" data-qa="card">
+      <img
+        src="/src/images/imac.jpeg"
+        alt="card picture imac"
+        class="card__image"
+      >
```

Comment:
We know it is iMac image, so let's give this information to search bots.

```suggestion
      <img
        src="/src/images/imac.jpeg"
        alt="APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)"
        class="card__image"
      >
```
![image](https://github.com/mate-academy/layout_product-cards/assets/96138503/cede408a-a968-49e9-af7e-c6d273565835)

https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules